start_control_ssm.sh

```diff
@@ -55,6 +67,8 @@ fetch_from_json() {
 # then check to see if the user-data file contains ssm at the top-level. If so,
 # attempt to manually register with SSM with a hybrid activation.
 
+link_host_certs
```

Comment:
```suggestion
[[ -d "${HOST_CERTS}" ]] && link_host_certs
```
nit, but it seems more clear to have the conditional outside the function.